Tests/Packet++Test/Tests/DoIpTests.cpp

```diff
@@ -0,0 +1,1146 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EndianPortable.h"
+#include "Packet.h"
+#include "EthLayer.h"
+#include "IPv4Layer.h"
+#include "TcpLayer.h"
+#include "UdpLayer.h"
+#include "PayloadLayer.h"
+#include "SystemUtils.h"
+#include "PacketUtils.h"
+#include "DeprecationUtils.h"
+#include <memory>
+#include <array>
+#include "DoIpLayer.h"
+#include "GeneralUtils.h"
+
+// DoIpRoutingActivationRequestPacketParsing
+PTF_TEST_CASE(DoIpRoutingActivationRequestPacketParsing)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/DoIpRoutingActivationRequestPacket.dat");
+
+	pcpp::Packet RoutingActivationRequestPacket(&rawPacket1);
```

Comment:
nit: the convention for variable names is to start with lower case. 
Please also fix variable names in the other tests below